.travis.yml

```diff
@@ -6,8 +6,10 @@ services:
 - docker
 script:
 - export REPO=mayadataio/dop-validator;
+- export PLAN=mayadataio/oep-e2eplan;
```

Comment:
we can define it as an env like Commit then we don't have to export twice.

Reply:
thanks @atulabhi Do you have any reference

Reply:
Done